src/WebApiContrib.Core.Formatter.Protobuf/ProtobufFormatterOptions.cs

```diff
@@ -0,0 +1,16 @@
+﻿
+using System.Collections.Generic;
+
+namespace WebApiContrib.Core.Formatter.Protobuf
+{
+    public class ProtobufFormatterOptions
+    {
+        //public IFormatterResolver FormatterResolver { get; set; } = ContractlessStandardResolver.Instance;
+
+        public HashSet<string> SupportedContentTypes { get; set; } = new HashSet<string> { "application/x-protobuf", "application/x-protobuf" };

```

Comment:
I think we should support all 3, opinion?
application/x-protobuf
application/protobuf
application/octet-stream